datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -60,15 +60,30 @@
   @VisibleForTesting
```

Comment:
Since you did the refactoring, can you please take of this as well ? May be move down after all private static final.

Reply:
done